assets/scss/modules/portfolio.scss

```diff
@@ -2,3 +2,17 @@
   background-color: $black-haze;
   min-height: 56.25rem;
 }
+
+.showcase {
+  padding-top: 5.625rem;
+}
+
+.img-list {
+  display: inline-block;
+  float: left;
+  list-style-type: none;
```

Comment:
Properties should be ordered display, float, height, list-style-type, margin, padding, width